config/webpack/webpack-dynamic-import.js

```diff
@@ -15,7 +24,9 @@ const prodConfig = {
     libraryTarget: 'umd',
     globalObject: 'window',
     filename: '[name].min.js',
-    publicPath: '/dicom-microscopy-viewer/'
```

Comment:
I think this should work - we might need to include the codec paths here too, but lets try it without